sdk/src/main/java/com/atlan/Atlan.java

```diff
@@ -20,8 +20,10 @@ public abstract class Atlan {
 
     // Note: these are set here so that they can be overridden,
     // i.e. when using the SDK in a workflow setting
-    public static final Map<String, List<String>> EXTRA_HEADERS =
-            Map.ofEntries(Map.entry("x-atlan-agent", List.of("sdk")), Map.entry("x-atlan-agent-id", List.of("java")));
+    public static final Map<String, List<String>> EXTRA_HEADERS = Map.ofEntries(
+            Map.entry("x-atlan-agent", List.of("sdk")),
+            Map.entry("x-atlan-agent-id", List.of("java")),
+            Map.entry("x-atlan-client-origin", List.of("product_sdk")));
```

Comment:
I see what you did here — makes general sense; however, these are really mean only for headers that can be overridden.

At least for now, we don't want `x-atlan-client-origin` to be overridden.

Can you look at adding it somewhere around here instead? 🙏  https://github.com/atlanhq/atlan-java/blob/fcdac5bc6777e3a7f5d7f20d7f74965ddec4b56a/sdk/src/main/java/com/atlan/net/HttpURLConnectionClient.java#L92